modules/govuk/manifests/apps/local_links_manager.pp

```diff
@@ -98,6 +98,11 @@
 #   The external subdomain used by the app
 #   Default: undef
 #
+# [*nagios_memory_warning*]
+#   The threshold that memory must be reached to be triggered as a warning
```

Comment:
Nitpick alternative comments! 

When an app's memory use reaches this value, a Nagios warning alert is triggered.
When an app's memory use reaches this value, a Nagios critical alert is triggered.

Reply:
Thanks, I took the comments from the asset_manager.pp manifest (they're also the same in the cache_clearing_service.pp manifest)